proto/eth/v2/shared.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 Prysmatic Labs.
```

Comment:
`shared.proto` feels too generic, I'd love to avoid dump it all file/folder (ie prysm/share/util, core/helpers) how about `version.proto`?